### PR TITLE
Add initial occollector information

### PR DESCRIPTION
**Pull request by flands**
EDIT: fixes https://github.com/census-instrumentation/opencensus-service/issues/39.

### CODE REVIEW

**Comment by flands** [comment]
FYI this has already been signed through Omnition

File: README.md [hunk 1]
```diff
@@ -186,4 +186,91 @@ backend (e.g Jaeger Thrift Span), and then push them to corresponding backend or
...
+* much more ...
+
+The collector also serves as a control plane for agents/clients by supplying
+them updated configuration (e.g. trace sampling policies), and reporting
+agent/client health information/inventory metadata to downstream exporters.
+
+### Architecture Overview
+
+The OpenCensus Collector runs as a standalone instance and receives spans and
+metrics exporterd by one or more OpenCensus Clients or Agents, or by
```
Comment (songy23) [inline README.md]: nit: Please replace "Client" with "Library", to be consistent with the previous section.

**Comment by flands** [comment]
@songy23 done, but looks like CLA needs to be added again.

@bogdandrutu please let me know if anything else is needed

**Comment by bogdandrutu** [comment]
If you signed the CLA as a company I think you need to use the company email (or at least have it as second email) in your github account.

**Comment by flands** [comment]
Thanks @bogdandrutu I did have the configured -- it appears the commit must have the company email (was set to personal). Works now. Change is ready for merge unless there is additional feedback.

File: README.md [hunk 2]
```diff
@@ -186,4 +186,91 @@ backend (e.g Jaeger Thrift Span), and then push them to corresponding backend or
...
+
+### Architecture Overview
+
+The OpenCensus Collector runs as a standalone instance and receives spans and
+metrics exporterd by one or more OpenCensus Libraries or Agents, or by
+tasks/agents that emit in one of the supported protocols. The Collector is
+configured to send data to the configured exporter(s). The following figure
+summarizes the deployment architecture:
+
+![OpenCensus Collector Architecture](image/collector-architecture.png "OpenCensus Collector Architecture")
```
Comment (bogdandrutu) [inline README.md]: I would change the picture to also show that the oc-agent can also fetch data from Zipkin and Jaeger.

**Comment by bogdandrutu** [comment]
@flands sorry for the delay. Some small comments.

File: README.md [hunk 3]
```diff
@@ -186,4 +186,91 @@ backend (e.g Jaeger Thrift Span), and then push them to corresponding backend or
 
 ## OpenCensus Collector
 
-TODO: add content about Collector.
+The OpenCensus Collector is a component that runs “nearby” (e.g. in the same
+VPC, AZ, etc.) a user’s application components and receives trace spans and
+metrics emitted by tasks instrumented with opencensus instrumentation (or other
```
[the remark this answers is not in the document]
Comment (flands) [reply README.md]: Makes sense -- done

File: README.md (hunk 2 above)
Comment (flands) [reply README.md]: The picture should already capture this -- the third example is Jaeger/Zipkin to OC Agent. Please let me know if you meant something else.

**Comment by flands** [comment]
@bogdandrutu  Updated -- let me know if anything else.

**Pull request merged**